.github/workflows/release.yaml

```diff
@@ -30,28 +30,32 @@ jobs:
         run: make build-release
       - name: Build and release Linux packages
         run: make build-linux-packages
+      - name: Set up QEMU
```

Comment:
Why do we need QEMU?

Reply:
If we want support for more platforms, we should QEMU with the Docker Setup QEMU action.
https://github.com/docker/setup-qemu-action